tests/testthat/test-encrypt-wrapper.R

```diff
@@ -30,10 +30,12 @@ test_that("nse 2", {
   filename <- tempfile()
   on.exit(file_remove_if_exists(filename))
```

Comment:
The test file could include "round trip" in the comments or test names, since current test names are a bit... cryptic :stuck_out_tongue_winking_eye: